app/controllers/user/birth_records_controller.rb

```diff
@@ -26,10 +26,8 @@ def query
   # Attempts to add a record which is already attached will be ignored (by
   # 'distinct' modifier on User.birth_records).
   def add
-    AuditedOperationsService.add_birth_record_to_user(
-      user: current_user,
-      birth_record: BirthRecord.find_by(params.permit(:id))
-    )
+    @birth_record = BirthRecord.find_by(params.permit(:id))
```

Comment:
I wanted to put this into a `before_action` but they actually all have different requirements - `#show` is looking specifically for the user's birth records and should throw an error if it doesn't find it, `#remove` is looking specifically for the user's birth records but shouldn't throw an error if it doesn't find it, and it doesn't need to already belong to the user in `#add`